SPA/services/getIntervieweesList.js

```diff
@@ -0,0 +1,20 @@
+let getIntervieweesList = async () => {
+    const options = {
+       method: 'GET',
+       headers: {
+           'Content-Type': 'application/json'
+       }
+   };
+   try {
+    const response = await fetch(`http://localhost:3000/interviewees/`, options)
+    const json = await response.json();
+    console.log("p")
+    console.log(json)
+    return json
+} catch (err) {
+ console.log("p")
```

Comment:
Remove redundant console logs please

Reply:
Thank you for reviewing. Sorry I didn't notice these, I will remove them.

Reply:
7bea17d49b222e81b0fb265f7ed1c09bf603163c